readme.md

```diff
@@ -241,15 +241,14 @@ and then run `dotnet build /t:GenerateCode` and it will attach a debugger to the
 
 To debug a TypeSpec project inside `azure-sdk-for-net` repo, you could run
 ```powershell
-dotnet build /t:GenerateCode /p:typespecAdditionalOptions="debug=true"
+dotnet build /t:GenerateCode /p:Debug=true
 ```
 to attach a debugger to the plugin process.
 
 If you are trying to use a locally built generator `dll` and debug it, you should run
 ```powershell
-dotnet build /t:GenerateCode /p:typespecAdditionalOptions="csharpGeneratorPath=/absolute/path/to/AutoRest.CSharp.dll%3Bdebug=true"
+dotnet build /t:GenerateCode /p:Debug=true /p:typespecAdditionalOptions="csharpGeneratorPath=/absolute/path/to/AutoRest.CSharp.dll"
 ```
-where you must use `%3B` as escape of the semicolon to append multiple options.
 
```

Comment:
Because of [this](https://github.com/Azure/autorest.csharp/pull/4970/files#r1708509245), we need to revert this because we still have to use the old way to debug the generator.